google/mozc
# Add/improve various symbols

## Description
A clear and concise description of this pull request.

Added/improved symbols composition:

- ~~合成用濁点・半濁点~~
- ダッシュ（em・en・水平線）
- 縦線（双柱・平行の区別）
- カッコ・引用符ペア
- 数学記号（±・∓・≠・≦・≧・三点リーダー）
- 将棋駒
- 麻雀牌
- 発音記号（英語用）
- 矢印（改行・斜め）
- 上付き・下付き文字

## Issue IDs
Issue and/or discussion IDs related to this pull request.

#929

## Steps to test new behaviors (if any)
A clear and concise description about how to verify new behaviors (if any).
 - OS: Any
 - Steps:
   1. Input
  2. Convert
  3. Confirm you can get the correct symbol

## Additional context
Add any other context about the pull request here.

I split commits according to symbol types.

- 麻雀牌
  - https://kinmaweb.jp/mahjong-rule/pai
  - https://mj-dragon.com/rule/basic/yougo.html
  - https://shop.taiyo-chemicals.co.jp/blog/?p=1288
  - https://media.jannavi.net/mahjong-sangenpai/
  - https://kotobank.jp/word/%E4%B8%89%E5%85%83%E7%89%8C-513317
  - https://ja.wikipedia.org/wiki/%E9%BA%BB%E9%9B%80%E7%89%8C
  - https://mjlocal.fandom.com/ja/wiki/%E7%99%BE%E6%90%AD
  - https://www.weblio.jp/content/%E4%B8%89%E5%85%83%E7%89%8C
- 将棋（先手後手）
  - https://将棋講座.com/%E5%B0%86%E6%A3%8B%E7%94%A8%E8%AA%9E/%E5%85%88%E6%89%8B.html
  - https://syougi-a.net/nyumon/8kifu.html
- 発音記号
  - https://www.kenkyusha.co.jp/purec/images/mihon%20hoka/327-76490-hatsuonkigo-namae.pdf
  - https://eigonokai.jp/phonetics/0-%e7%99%ba%e9%9f%b3%e8%a8%98%e5%8f%b73%e6%99%82%e9%96%93%e3%83%9e%e3%82%b9%e3%82%bf%e3%83%bc/
